recipes-sota/rvi-sota-client/rvi-sota-client_git.bb

```diff
@@ -11,13 +11,13 @@ S = "${WORKDIR}/git"
 # When changing this, don't forget to:
 # 1) Update PV
 # 2) Check that Cargo.lock hasn't changed with git diff old..new Cargo.lock
-SRCREV = "b6cf6957203fc406b3723d046f1b705e1bd08d7d"
+SRCREV = "ee293e663c2eabe7d9fe96934c60db5d8c71fc80"
```

Comment:
It looks like Cargo.lock has changed between these versions: do you need to bump the version numbers in other crates in this recipe?

(Or is that in the other PR?  I'm just about to look at that one..)